src/app/legacy/containers/MediaPlayer/helpers/propsInference/default/index.js

```diff
@@ -29,6 +30,12 @@ export default ({
     path(['model', 'blocks'], aresMediaBlock),
     'caption',
   );
+  const hasWebcastVersions = prop(
```

Comment:
Just a quick question - what is the difference between `prop` and `path`?

Reply:
It's basically the same thing, but prop doesn't take an array as a parameter, so only queries the top level props instead of going deeper into the nested objects. Eg.

`prop(x, {x: {y:0} } ) = 0`, 

but we couldn't do this:

`prop(['x','y'], {x: {y:0} } ) = undefined` as prop doesn't take an array as a parameter.

Reply:
Example [here](https://ramdajs.com/repl/?v=0.29.1#?R.path%28%5B%27x%27%2C%27y%27%5D%2C%20%7Bx%3A%7By%3A%20100%7D%7D%29%3B%20%2F%2F%3D%3E%20100%C2%A0%0AR.prop%28%5B%27x%27%2C%27y%27%5D%2C%20%7Bx%3A%7By%3A%20100%7D%7D%29%3B%20%2F%2F%3D%3E%20undefined%C2%A0%0AR.prop%28%27x%27%2C%20%7Bx%3A%7By%3A%20100%7D%7D%29%3B%20%2F%2F%3D%3E%20%7B%22y%22%3A%20100%7D%C2%A0)

Reply:
Thanks for the explanation and the example (although I have to admit, that example code makes me 🤯 ). Ramda is "good" for stuff like this, but just awful to read sometimes 😅